static/less/richtext.less

```diff
@@ -215,21 +214,21 @@
         vertical-align: text-top;
     }
 
-    .ProseMirror-editor-source{
+    .ProseMirror-editor-source {
         border: 2px solid #ededed;
         border-radius: 0 4px 4px 4px;
         box-sizing: border-box;
         display: block;
         min-height: 36px;
-        resize: vertical;
+        resize: none;
         overflow: auto;
         outline: none;
         padding: 7px 12px;
         width: 100%;
     }
 
     .ProseMirror-editor-source:focus {
-        border: 2px solid #0b97c4;
+        border: 2px solid #21A1B3;
```

Comment:
@yuriimaz Shouldn't we use a color var here?

Reply:
@luke- Done. I changed all color hash codes in editor less files to variables.